ss-tproxy

```diff
@@ -939,7 +939,7 @@ start_iptables_tproxy_mode() {
 start_iptables_redirect_mode() {
     local loopback_addr
     is_ipv4_ipts $1 && loopback_addr="127.0.0.1" || loopback_addr="::1"
-    
+
```

Comment:
You pay attention to details, however, these spaces have been removed in my recent commit, thanks for the reminder.